ci_tools/flake8-requirements.txt

```diff
@@ -1,6 +1,6 @@
 setuptools_scm>=3,<4
 flake8>=3.6,<4
-flake8-html>=0.4,<1
```

Comment:
You can revert this actually: this file is only used in the nox session named `flake8`, and this session is not executed by CI.